src/resources/LibraryStyles.css

```diff
@@ -397,7 +398,6 @@ input {
 .SearchResultItemContainer .ItemIcon {
     padding: 2px 8px;
     width: 40px;
-    height: 40px;
```

Comment:
Yup I realised this. However, the width of `.ItemInfo` will be changed when the description text is shown, which will change the width of `.ItemIcon`, and that's why I set the width of `.ItemInfo` to be 100% so that its width stays unchanged. To keep the native dimension of the icon, I tried that set the width of `.ItemInfo` to be 90% works, and it doesn't affect the dimension of the icon. Is this fine?